combat-trainer.lic

```diff
@@ -1097,7 +1099,13 @@ class TrainerProcess
     when 'Meraud'
       meraud_commune(game_state)
     when 'Recall'
-      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?
+      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?   
+    when 'Research Augmentation'
+      bput('meditate research monkey', 'You clear ') if DRStats.barbarian?
```

Comment:
So then do we add a validate for it? A lot of times people will just copy and paste things, and this will just spam and spit out errors if we don't.